fleet/api/http/transport.go

```diff
@@ -90,6 +90,11 @@ func MakeHandler(tracer opentracing.Tracer, svcName string, svc fleet.Service) h
 		decodeListBackends,
 		types.EncodeResponse,
 		opts...))
+	r.Get("/agents/:id/reset", kithttp.NewServer(
```

Comment:
let's make the endpoint /agents/:id/rpc/reset and make it a POST instead of a GET (because GET usually means "read only")